### PR TITLE
Newcourses

### PR DESCRIPTION
**Pull request by ravi0900**
## What does this PR do?
Add Hindi and English Course resources 

### Description
* Added hindi Javascript and HTML/CSS course
* Added English in depth C and Data Structure Playlist

 

### Why is this valuable (or not)?
* this is valuable because these cover the topic that not covered in the previous mentioned courses and which is important to know as a programmer,

### How do we know it's really free?
All the resources are shared via youtube URL, so it's free to watch
### For book lists, is it a book? For course lists, is it a course? etc.
It is a course list

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by ravi0900** [comment]
Please merge my contribution, thankyou.
#hacktoberfest

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -223,6 +223,8 @@
 
 ### C
 
+* [C & Data Structure \| Depth understanding](https://www.youtube.com/watch?v=4OGMB4Fhh50&list=PLBlnK6fEyqRhX6r2uhhlubuF5QextdCSM) - Neso Academy
```
Comment (eshellman) [inline courses/free-courses-en.md:226]: For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -223,6 +223,8 @@
...
+* [C | A better Understanding](https://www.youtube.com/watch?v=rLf3jnHxSmU&list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR) - Neso Academy
```
Comment (eshellman) [inline courses/free-courses-en.md:227]: For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by ravi0900** [comment]
I committed change request.

**Comment by ravi0900** [comment]
@eshellman this is my first ever hactoberfest attempt, please merge this request and label as hacktoberfest-accepted
thank you.

File: courses/free-courses-en.md (hunk 2 above)
Comment (ravi0900) [reply courses/free-courses-en.md]: I made the changes

File: courses/free-courses-en.md (hunk 1 above)
Comment (ravi0900) [reply courses/free-courses-en.md]: I made the change

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -223,6 +223,8 @@
 
 ### C
 
+* [C & Data Structure \| Depth understanding](https://www.youtube.com/watch?v=4OGMB4Fhh50&list=PLBlnK6fEyqRhX6r2uhhlubuF5QextdCSM) - Sujeet Singh(Neso Academy)
```
Comment (davorpa) [inline courses/free-courses-en.md:226]: - Use titles as it is
- Use YT playlist urls syntax

```suggestion
* [C Programming & Data Structures](https://www.youtube.com/playlist?list=PLBlnK6fEyqRhX6r2uhhlubuF5QextdCSM) - Sujeet Singh (Neso Academy)
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -223,6 +223,8 @@
...
+* [C | A better Understanding](https://www.youtube.com/watch?v=rLf3jnHxSmU&list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR) - Sujeet Singh(Neso Academy)
```
Comment (davorpa) [inline courses/free-courses-en.md:227]:
```suggestion
* [C Programming](https://www.youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR) - Sujeet Singh (Neso Academy)
```

File: courses/free-courses-hi.md [hunk 5]
```diff
@@ -119,7 +119,9 @@
 
 ### HTML and CSS
 
+* [CSS Mastery in one Video 9 hours](https://www.youtube.com/watch?v=WyxzAU3p8CE&t=6063s) - Vishwajeet Kumar(Youtube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:122]: - Using titles as same as the provided by the original resource
- Removing start timestamp from YT url
- Use syntax for instructors
- No YT annotations

```suggestion
* [CSS Tutorial in Hindi \| Complete CSS Course For Beginners to Advanced \| Step By Step Tutorial](https://www.youtube.com/watch?v=WyxzAU3p8CE) - Tech Gun, `tch.:` Vishwajeet Kumar
```

File: courses/free-courses-hi.md [hunk 6]
```diff
@@ -119,7 +119,9 @@
...
 * [HTML CSS & Browser APIs](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC) - Pepcoding
+* [HTMl in one Video 4 hours](https://www.youtube.com/watch?v=QXPWs00RD3A&t=5884s) - Vishwajeet Kumar(youtube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:124]:
```suggestion
* [HTML Tutorial in Hindi \| Complete HTML Course For Beginners to Advanced](https://www.youtube.com/watch?v=QXPWs00RD3A) - Tech Gun, `tch.:` Vishwajeet Kumar
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Please, apply review suggestions

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

**Comment by ravi0900** [comment]
> Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

Ohfo I'm not able to fix this linter error, can you help with that, I mean everything seems fine in code but linter check failed ! 
build error : -
/home/runner/work/free-programming-books/free-programming-books/courses/free-courses-en.md
  230:1-236:92  warning  Alphabetical ordering: swap l.232 and l.231  alphabetize-lists  remark-lint

**Comment by davorpa** [comment]
> Ohfo I'm not able to fix this linter error, can you help with that, I mean everything seems fine in code but linter check failed ! 

I think now is all in order 😉 . Sorry for the inconvenients

**Comment by ravi0900** [comment]
> > Ohfo I'm not able to fix this linter error, can you help with that, I mean everything seems fine in code but linter check failed !
> 
> I think now is all in order wink . Sorry for the inconvenients

Ya 😄 thank god ! Sorry I'm newbie so made too many commits, this is my first time participation in hactoberfest, please merges with hacktoberfest-accepted label, thank you.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**